### PR TITLE
Feature/add sound recording work type

### PR DESCRIPTION
**Pull request by ewilloughby**
Hey Brandon - this PR adds the sound recording work type.  I tested uploading an mp3 - the player does appear once fits has done its thing and the only unique field for this type - extent - displays.  I also updated a few labels to match the wording on our other similar tests.

### CODE REVIEW

File: spec/features/admin_user/edit_works/admin_user_can_edit_sound_recording_spec.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		before do
+			# Make admin role and make user admin
+			admins = Role.create(name: 'admin')
+			admins.users << user
+
+			login_as user
+		end
+
+		scenario 'An admin user can edit a sound recording they created' do
+			visit edit_hyrax_artifact_path(artifact.id)
```
Comment (boveus) [inline spec/features/admin_user/edit_works/admin_user_can_edit_sound_recording_spec.rb]: This part should be edit_hyrax_sound_recording_path

File: spec/features/admin_user/edit_works/admin_user_can_edit_sound_recording_spec.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+			fill_in('Title', with: 'New title')
+			fill_in('Identifier', with: 'testing')
+			click_on('Save')
+			expect(page).to have_content('New title')
+		end
+
+		scenario 'An admin user can edit a sound recording someone else created' do
+			visit edit_hyrax_artifact_path(other_user_artifact.id)
```
Comment (boveus) [inline spec/features/admin_user/edit_works/admin_user_can_edit_sound_recording_spec.rb]: Same as above.

File: spec/features/metadata/sound_recording/admin_can_create_a_sound_recording_with_all_metadata_spec.rb [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+      expect(page).to have_content( 'This is a test item being created for a sound recording')
+      expect(page).to have_content( 'English')
+      expect(page).to have_content( 'Testing the note field')
+      expect(page).to have_content( 'Series 242342')
+      expect(page).to have_content( 'Southern Labor Archives')
+      expect(page).to have_content( 'some finding aid')
+      expect(page).to have_content( 'citation')
+      expect(page).to have_content( 'Georgia State University Library')
+      expect(page).to have_content( 'audio/mpeg')
+      expect(page).to have_content( 'Audio')
```
Comment (boveus) [inline spec/features/metadata/sound_recording/admin_can_create_a_sound_recording_with_all_metadata_spec.rb]: Oh, it looks like you are correct.  If we change source format or item type we may need to refactor these tests as well as the Model tests.

File: spec/features/admin_user/edit_works/admin_user_can_edit_sound_recording_spec.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+		let(:user_attributes) do
+			{ email: 'test@example.com' }
+		end
+
+		let(:user) do
+			User.new(user_attributes) { |u| u.save(validate: false) }
+		end
+
+		let(:artifact) do
+			Artifact.create(depositor: user.user_key,
```
Comment (boveus) [inline spec/features/admin_user/edit_works/admin_user_can_edit_sound_recording_spec.rb]: This should be a sound recording

File: spec/features/metadata/sound_recording/admin_can_create_a_sound_recording_with_all_metadata_spec.rb (hunk 3 above)
Comment (ewilloughby) [reply spec/features/metadata/sound_recording/admin_can_create_a_sound_recording_with_all_metadata_spec.rb]: Hey Brandon - thanks for catching that!  Apparently I didn't save all of my changes to this file before pushing. I've fixed the test so it references sound recording and not artifact!

**Pull request merged**